controllers/grid/queries/form/QueryForm.php

```diff
@@ -319,13 +319,11 @@ public function fetch($request, $template = null, $display = false, $actionArgs
             // if current user is an anonymous reviewer, filter out authors
             foreach ($reviewAssignments as $reviewAssignment) {
                 if ($reviewAssignment->getReviewerId() == $user->getId()) {
-                    if ($reviewAssignment->getReviewMethod() != ReviewAssignment::SUBMISSION_REVIEW_METHOD_OPEN) {
-                        // Replaces StageAssignmentDAO::getBySubmissionAndRoleId
+                    if ($reviewAssignment->getReviewMethod() == ReviewAssignment::SUBMISSION_REVIEW_METHOD_DOUBLEANONYMOUS ) {
+                        // exclude authors from the list of participants to ensure anonymity
                         $excludeUsers = StageAssignment::withSubmissionIds([$query->getAssocId()])
                             ->withRoleIds([Role::ROLE_ID_AUTHOR])
-                            ->withUserId($user->getId())
-                            ->get()
-                            ->pluck('userId')
+                            ->pluck('user_id')
```

Comment:
We avoid usage of column names outside of a query builder. I see that UseCamelCasing doesn't convert when not calling the model directly. If it's impossible to avoid, we can just create a scope for a Model. Here we can just get the collection first and then call pluck on the collection, how it was implemented before.